spec/models/sub_section_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe SubSection do
+  let(:sub_section) { create :sub_section }
+
+  describe "validations" do
+    it "should belong to a coronavirus_page" do
+      should validate_presence_of(:coronavirus_page)
```

Comment:
Personally, I'm not a great fan of shoulda matchers - they usually shorthand specs for stuff that doesn't really need testing (like joins). However, as they are present, if you want to test `validate :presence` then the whole of this spec could be replaced with:

```ruby
RSpec.describe SubSection, , type: :model do
  it "validates presence" do
    should validate_presence_of(:coronavirus_page)
    should validate_presence_of(:title)
    should validate_presence_of(:content)
  end
end
```